diff -ur ibus-table-chinese-1.3.1-Source.orig/CMakeLists.txt ibus-table-chinese-1.3.1-Source/CMakeLists.txt
--- ibus-table-chinese-1.3.1-Source.orig/CMakeLists.txt	2011-06-22 05:19:30.000000000 +0200
+++ ibus-table-chinese-1.3.1-Source/CMakeLists.txt	2011-07-20 22:08:29.742699912 +0200
@@ -23,28 +23,6 @@
 SET(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/Modules ${CMAKE_ROOT}/Modules )
 MESSAGE("CMAKE_MODULE_PATH=${CMAKE_MODULE_PATH}")
 
-INCLUDE(ManageVariable RESULT_VARIABLE MANAGE_VARIABLE)
-IF(MANAGE_VARIABLE STREQUAL "NOTFOUND")
-    MESSAGE(FATAL_ERROR "ManageVariable is not found in CMAKE_MODULE_PATH,
-    please:
-    1) install cmake-fedora, or;
-    2) wget https://fedorahosted.org/releases/c/m/cmake-fedora/cmake-fedora-modules-only-latest.tar.gz; tar zxvf cmake-fedora-modules-only-latest.tar.gz")
-ENDIF()
-
-IF(CMAKE_VERSION)
-    CMAKE_POLICY(SET CMP0005 OLD)
-ENDIF(CMAKE_VERSION)
-
-INCLUDE(ManageVersion)
-LOAD_RELEASE_FILE("${CMAKE_SOURCE_DIR}/RELEASE-NOTES.txt")
-INCLUDE(UseUninstall)
-
-INCLUDE(CompileEnv)
-SET_USUAL_COMPILE_ENVS()
-
-INCLUDE(CMakeVersion)
-
-
 ####################################################################
 # Required
 #
@@ -68,19 +46,11 @@
 
 GET_FILENAME_COMPONENT(_releaseFileName "${RELEASE_FILE}" NAME)
 SET(INSTALL_DOCS ${_releaseFileName} AUTHORS README ChangeLog COPYING README)
-STRING_JOIN(PRJ_DOC_LIST " " ${INSTALL_DOCS})
 
-INSTALL(FILES ${INSTALL_DOCS}
-    DESTINATION "${PRJ_DOC_DIR}")
 
 ####################################################################
 # Packing
 #
-SET(MAINTAINER_SETTING "MAINTAINER_SETTING_NO_PACK")
-INCLUDE(PackSource)
-INCLUDE(PackRPM)
-INCLUDE(ManageReleaseOnFedora)
-
 SET(PRJ_IGNORE_FILES_COMMON
     "\\\\.spec$" "messages.po$" "\\\\.orig$" "\\\\.db$"
     "/\\\\.cache/" "/Modules/"
@@ -94,29 +66,6 @@
 LIST(APPEND PACK_SOURCE_IGNORE_FILES
     ${PRJ_IGNORE_FILES_COMMON} "/bin/")
 
-PACK_SOURCE(PACK_SOURCE_FILE_NAME "${RPM_BUILD_SOURCES}")
-PACK_RPM(PRJ_SRPM_FILE_NAME "${RPM_BUILD_SPECS}/project.spec.in"
-    "${PACK_SOURCE_FILE_NAME}")
-
-# If mock exists, then add mock related targets.
-FIND_PROGRAM(MOCK_CMD mock)
-IF(NOT MOCK_CMD STREQUAL "MOCK_CMD-NOTFOUND")
-    USE_MOCK("${CMAKE_SOURCE_DIR}/SPECS/project.spec.in")
-ENDIF(NOT MOCK_CMD STREQUAL "MOCK_CMD-NOTFOUND")
-
-IF(EXISTS ${MAINTAINER_SETTING})
-    RELEASE_ON_FEDORA("${RPM_BUILD_SRPMS}/${PRJ_SRPM_FILE_NAME}"
-	${FEDORA_CURRENT_RELEASE_TAGS})
-ENDIF(EXISTS ${MAINTAINER_SETTING})
-
-####################################################################
-# Hosting
-#
-INCLUDE(ManageMaintainerTargets)
-MAINTAINER_SETTING_READ_FILE("${MAINTAINER_SETTING}"
-    "${RPM_BUILD_SOURCES}/${PACK_SOURCE_FILE_NAME}")
-
-
 ####################################################################
 # Test Suites.
 #
diff -ur ibus-table-chinese-1.3.1-Source.orig/tables/CMakeLists.txt ibus-table-chinese-1.3.1-Source/tables/CMakeLists.txt
--- ibus-table-chinese-1.3.1-Source.orig/tables/CMakeLists.txt	2011-06-22 05:19:30.000000000 +0200
+++ ibus-table-chinese-1.3.1-Source/tables/CMakeLists.txt	2011-07-20 22:12:14.603170183 +0200
@@ -4,7 +4,7 @@
 ENDIF(CREATE_DB STREQUAL "CREATE_DB-NOTFOUND")
 
 IF(NOT DEFINED DB_INSTALL_DIRECTORY)
-    SET(DB_INSTALL_DIRECTORY "${DATA_DIR}/ibus-table/tables")
+	SET(DB_INSTALL_DIRECTORY "${SHARE_INSTALL_PREFIX}/ibus-table/tables")
 ENDIF(NOT DEFINED DB_INSTALL_DIRECTORY)
 
 IF(NOT DEFINED DEFAULT_FREQ)
@@ -12,7 +12,7 @@
 ENDIF(NOT DEFINED DEFAULT_FREQ)
 
 IF(NOT DEFINED ICON_INSTALL_DIRECTORY)
-    SET(ICON_INSTALL_DIRECTORY "${DATA_DIR}/ibus-table/icons")
+	SET(ICON_INSTALL_DIRECTORY "${SHARE_INSTALL_PREFIX}/ibus-table/icons")
 ENDIF(NOT DEFINED ICON_INSTALL_DIRECTORY)
 
 FIND_PROGRAM(SED_CMD sed)
